mountinfo/mountinfo_linux_test.go

```diff
@@ -504,66 +499,6 @@ func TestParseFedoraMountinfoFilterFields(t *testing.T) {
 	}
 }
 
-func TestParseMountinfoWithSpaces(t *testing.T) {
```

Comment:
I don't understand how removing this test is relevant to this commit.

Reply:
To reiterate the comment above -- this test case is NOT about pre-3.10 bug of having extra spaces in a cifs mount entry where share name has spaces.

Reply:
@thaJeztah ^^^ (or do we have other tests where we test fields with spaces)?

Reply:
>  this test case is NOT about pre-3.10 bug of having extra spaces in a cifs mount entry where share name has spaces.

Oh! Right, only the second entry is about the `unc=<path with spaces>` issue; the others are escaped

IIUC from the comment, kernel 3.10-rc1 should ignore that `unc` option, so I can just remove that part.

Do we know of other ways for spaces to get into the last column, or should  3.10-rc1 and up guarantee there not to be?

> (or do we have other tests where we test fields with spaces)?

@kolyshkin hm.. not sure if we have; I'll add back this test, but update it to only remove the `unc=` bits.